packages/ckeditor5-engine/src/model/selection.ts

```diff
@@ -637,13 +637,23 @@ export default class Selection extends EmitterMixin( TypeCheckable ) {
 	 * </block>
 	 * ```
 	 *
-	 * **Special case**: If a selection ends at the beginning of a block, that block is not returned as from user perspective
-	 * this block wasn't selected. See [#984](https://github.com/ckeditor/ckeditor5-engine/issues/984) for more details.
+	 * **Special case**: Selection ignores first and/or last blocks if nothing (from user perspective) is selected in them.
 	 *
 	 * ```xml
+	 * // Selection ends and the beginning of the last block
 	 * <paragraph>[a</paragraph>
 	 * <paragraph>b</paragraph>
-	 * <paragraph>]c</paragraph> // this block will not be returned
+	 * <paragraph>]c</paragraph> // This block will not be returned
```

Comment:
My OCD: This comment start from an upper case letter, while later on the same comment (different function) starts with a lower case :D